### PR TITLE
Using hash auto generation of IDs instead of sequences

### PR DESCRIPTION
**Pull request by vkorobkov**
https://github.com/vkorobkov/jfixtures/issues/64

Using IDs generate by row alias hashes instead of sequences

### CODE REVIEW

File: src/main/java/com/github/vkorobkov/jfixtures/processor/Processor.java [hunk 1]
```diff
@@ -83,8 +84,8 @@ private Instruction processRow(String tableName, FixtureRow row) {
         Map<String, FixtureValue> result = new LinkedHashMap<>(row.getColumns().size() + 1);
         val table = context.getConfig().table(tableName);
         if (table.shouldAutoGeneratePk()) {
-            result.put(table.getPkColumnName(),
-                    context.getSequenceRegistry().nextValue(tableName, row.getName()));
+            FixtureValue id = new FixtureValue(Id.one(row.getName()));
```
Comment (rodionovsasha) [inline src/main/java/com/github/vkorobkov/jfixtures/processor/Processor.java]: val id?

File: src/test/groovy/com/github/vkorobkov/jfixtures/AfterInsertsTest.groovy [hunk 2]
```diff
@@ -5,10 +5,10 @@ import spock.lang.Specification
 
 class AfterInsertsTest extends Specification implements YamlVirtualFolder {
     def DEFAULT_EXPECTED_SQL = """DELETE FROM "users";
-            |INSERT INTO "users" ("id", "name", "age") VALUES (10000, 'Vladimir', 29);
+            |INSERT INTO "users" ("id", "name", "age") VALUES (${Id.one('vlad')}, 'Vladimir', 29);
```
Comment (rodionovsasha) [inline src/test/groovy/com/github/vkorobkov/jfixtures/AfterInsertsTest.groovy:8]: :+1:

Comment (rodionovsasha) [reply src/test/groovy/com/github/vkorobkov/jfixtures/AfterInsertsTest.groovy:8]: does it have backward compatibility?

**Review by rodionovsasha — APPROVED** [review]

File: src/test/groovy/com/github/vkorobkov/jfixtures/AfterInsertsTest.groovy (hunk 2 above)
Comment (vkorobkov) [reply src/test/groovy/com/github/vkorobkov/jfixtures/AfterInsertsTest.groovy:8]: What do you mean?

**Pull request merged**